base/_typography.scss

```diff
@@ -238,11 +232,172 @@ table {
 	}
 
 	// zebra striping pattern
-	&.striped {
-		tbody {
-			tr:nth-of-type(odd) {
-				background-color: $table-border-color;
-			}
+	tbody {
+
+		tr:nth-of-type(odd) {
+			background-color: $table-border-color;
+		}
+	}
+}
+
+// Inline typography elements
+strong,
+b,
+em,
+i,
+cite,
+small,
+sup,
+sup { letter-spacing: .03125em; }
+
+sub,
+sup {
+	font-size: 75%;
+	line-height: 1;
+	position: relative;
+}
+
+sup { top: -0.5em; }
+
+sub { bottom: -0.25em; }
+
+abbr[title],
+cite,
+dfn[title] { border-bottom: 1px dotted palette(brand); }
+
+abbr[title],
+dfn[title] { cursor: help; }
+
+abbr[title] {
+	font-size: 75%;
+	font-weight: 700;
+	letter-spacing: 0.125em;
+	text-transform: uppercase;
+}
+
+b,
+strong { font-weight: 700; }
+
+i,
+em,
+cite { font-style: italic; }
+
+var {
+	color: palette(mono, 75);
+	display: inline-block;
+	font-style: italic;
+	padding: 0 .125em;
+}
+
+// Proper code blocks need to be surrounded by a <pre>. They work together.
+pre {
+	// scss-lint:disable ImportantRule
+	white-space: pre-wrap;
+	white-space: -moz-pre-wrap !important;	// 1999+ Mozilla
+	white-space: -o-pre-wrap;				// Opera 7 and up
+	word-wrap: break-word;					// IE 5.5+ and up
+
+	code {
+		border-bottom: 1px solid palette(mono, 25);
+		border-left: 5px solid palette(brand, green);
+		border-radius: 0;
+		display: block;
+		margin: 0;
+		padding: 1.375em 1.25em 1.3125em;
+	}
+}
+
+code,
+kbd,
+samp,
+mark,
+ins {
+	display: inline-block;
+	line-height: 1.5;
+	padding: .125em .25em 0;
+}
+
+code,
+kbd,
+samp {
+	// scss-lint:disable DuplicateProperty
+	background: palette(mono, 10);
+	background: rgba(#000, .05);
+	border-radius: 4px;
+	box-shadow: 0 0 .25em rgba(#000, .1) inset;
+	font-family: $font-mono;
+}
+
+code {
+	color: palette(brand, green);
+}
+
+mark { background-color: #ff9; }
+
+strike,
+s { color: palette(mono, 25); }
+
+del { color: palette(brand); }
+
+u { text-decoration: underline; }
+
+ins {
+	background-color: palette(mono, 25);
+	text-decoration: none;
+}
+
+small { font-size: 87.5%; }
+
+// These should change depending on the language
+[lang="en-US"] {
+
+	q {
+		display: inline;
+
+		::before { content: "“"; }
+
+		::after {
+			content: "”";
+			white-space: nowrap;
 		}
+
+		// Nested
+		q::before { content: "‘"; }
+
+		q::after { content: "’"; }
 	}
 }
+
+.typography {
```

Comment:
Style guide. We could use it, but best to apply any specific container classes to the vertical rhythm, which is now inside `_content.scss` as '.content--main { etc }`